pandas/_libs/tslibs/period.pyx

```diff
@@ -1305,6 +1305,24 @@ cdef class _Period(object):
 
     @property
     def week(self):
+        """
+        Get the total weeks on the Period falls in.
```

Comment:
Total weeks -> "week of year"?

We'll want to keep an eye on https://github.com/pandas-dev/pandas/issues/20324 for this.

Reply:
"Get the week of the year on the given Period"?

sure will check #20324

Reply:
Can you update the summar?